master-firmware/protobuf/strategy.proto

```diff
@@ -11,18 +11,18 @@ enum PuckColor {
     RED = 0;
     GREEN = 1;
     BLUE = 2;
-    WHITE = 3;
+    REDGREEN = 3;
 }
 
 message RobotState {
     option (nanopb_msgopt).msgid = 14;
```

Comment:
I would suggest adding `option (nanopb_msgopt).packed_struct = true;` to RobotState. Using that and reordering the fields to have all the `bool` together (for optimal packing), I got the state size from 32 to 26 bytes. Might not seem much, but every edge matters.